### PR TITLE
ref(ui): Remove unneeded mediaUrl

### PR DESCRIPTION
**Pull request by evanpurkhiser**
This is the only usage of this, better to just let webpack handle this stuff

### CODE REVIEW

File: src/sentry/static/sentry/app/components/resourceCard.jsx [hunk 1]
```diff
@@ -3,15 +3,14 @@ import PropTypes from 'prop-types';
...
-import ConfigStore from 'app/stores/configStore';
 import ExternalLink from 'app/components/links/externalLink';
 import {Panel} from 'app/components/panels';
 
 export default class ResourceCard extends React.Component {
   static propTypes = {
     title: PropTypes.string.isRequired,
     link: PropTypes.string.isRequired,
-    imgUrl: PropTypes.string.isRequired,
+    imgUrl: PropTypes.oneOfType([PropTypes.string, PropTypes.object]).isRequired,
```
Comment (BYK) [inline src/sentry/static/sentry/app/components/resourceCard.jsx]: Why this type change? Also this makes it unsafe on line 28 below.

File: src/sentry/static/sentry/app/views/organizationProjectsDashboard/resources.jsx [hunk 2]
```diff
@@ -9,6 +9,10 @@ import ErrorRobot from 'app/components/errorRobot';
 import {Panel} from 'app/components/panels';
 import {t} from 'app/locale';
 
+import releasesImg from '../../../images/releases.svg';
```
Comment (BYK) [inline src/sentry/static/sentry/app/views/organizationProjectsDashboard/resources.jsx]: we can probably add an alias for `images/` so these become `images/releases.svg` etc.

**Review by BYK — APPROVED** [review]
Just concerned about the type change.

File: src/sentry/static/sentry/app/components/resourceCard.jsx (hunk 1 above)
Comment (EvanPurkhiser) [reply src/sentry/static/sentry/app/components/resourceCard.jsx]: Yeah.. In our tests we do this:

https://github.com/getsentry/sentry/blob/5dbc8befd07244fa788a39b1f9aa348493876ee6/jest.config.js#L14

which exports an object. So when tests run on this it recieves an object and fails the test due to the type error :( I spent a bit of time trying to figure out how to fix. I'll add a comment describing why we did it

File: src/sentry/static/sentry/app/views/organizationProjectsDashboard/resources.jsx (hunk 2 above)
Comment (EvanPurkhiser) [reply src/sentry/static/sentry/app/views/organizationProjectsDashboard/resources.jsx]: Yeah we should. I'm not worried about it right now though, we can clean this up later

File: src/sentry/static/sentry/app/components/resourceCard.jsx (hunk 1 above)
Comment (BYK) [reply src/sentry/static/sentry/app/components/resourceCard.jsx]: I'll try to fix this.

**Pull request merged**